_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -9,84 +9,308 @@ nav_order: 50
-- Must have an X-axis on the bottom
-- Must define one X-axis aggregation bucket
-- Must have a valid time-based X-axis
+You can only create [per query and per bucket monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/per-query-bucket-monitors/) from within visualizations.
+{: .note}
 
-## Creating alerting monitors
+### Creating monitors
 
-By default, when you begin to create the alert monitor workflow using the Dashboard interface, you are presented with a menu-driven interface. This interface provides a range of options that are displayed in full screen, in pop-ups, in pull-downs, or in dropdowns. They allow you to define the metrics that can be monitored, set thresholds, customize triggers that automate workflows, and generate actions when conditions are met. Currently, you can only create [per query monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/monitors/).
+Follow these steps to create an alerting monitor:
 
-To create an alerting monitor: 
+1. Before creating a new monitor, save your dashboard. If you are unfamiliar with saving dashboards, see [Saving dashboards]({{site.url}}{{site.baseurl}}/dashboards/dashboard/index/#saving-dashboards).
+2. From your dashboard, select the visualization from which you want to create an alert. 
+3. Select the {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/} icon on the upper-right of the visualization and then select **Alerting** from the **Options** dropdown menu, as shown in the following image.
 
-1. Choose **Dashboard** from the OpenSearch Dashboards main menu.
-2. From the **Dashboards** window, select **Create** and then choose **Dashboard**.
-3. Select **Add an existing**, then select the appropriate alerting visualization from the **Add panels** list. The visualization is added to the dashboard.
-4. From the visualization panel, choose the ellipsis icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/}). 
-5. From the **Options** menu, select **Add alerting monitor**.
-6. Input information for **Monitor details** and **Triggers**.
-7. Choose **Create monitor**. The monitor is added to the visualization.  
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/dashboard-options-context-menu.png" alt="Selecting Alerting from Options dropdown menu within a visualization" width="700"/>
 
-An example of these steps is shown in the following screenshot.
+4. From the **Alerting** dropdown menu, select **Add alerting monitor**, as shown in the following image.
 
-<img src="{{site.url}}{{site.baseurl}}/images/dashboards/create-monitor-menu.png" alt="Create monitor interface" width="400" height="400">
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/add-monitor.png" alt="Adding an alerting monitor from within a visualization" width="700"/>
```

Comment:
Should we box only alerting in red and not both alerting and anomaly detection?

Reply:
Revised

Reply:
Removed images per writing style guidelines.